service/src/main/java/aero/minova/cas/controller/SqlProcedureController.java

```diff
@@ -209,6 +186,56 @@ public ResponseEntity executeProcedure(@RequestBody Table inputTable) throws Exc
 		}
 	}
 
+	/**
+	 * Überprüft, ob es für den Namen der übergebenen Table einen passenden Eintrag in den Extensions gibt und gibt das Ergebnis der ausgeführten Extension als
+	 * ResponseEntity zurück.
+	 * 
+	 * @param inputTable
+	 *            Eine Table. Muss einen Namen haben.
+	 * @return Das Ergebnis der Extension mit der übergebenen Table als Input.
+	 */
+	private ResponseEntity checkForExtension(Table inputTable) {
+		if (extensions.containsKey(inputTable.getName())) {
+			synchronized (extensionSynchronizer) {
```

Comment:
Am besten containsKey mit synchronisieren, falls in Zukunft-Extensions schon beim Start vom CAS ausgeführt werden.